tests/sentry/models/test_organizationmember.py

```diff
@@ -202,3 +202,19 @@ def test_scopes_with_member_admin_config(self):
         organization.update_option("sentry:events_member_admin", False)
 
         assert "event:admin" not in member.get_scopes()
+
+    def test_scopes_with_member_alert_write(self):
+        organization = self.create_organization()
+        member = OrganizationMember.objects.create(
+            organization=organization, role="member", email="test@example.com",
+        )
+
+        assert "alerts:write" in member.get_scopes()
+
+        organization.update_option("sentry:alerts_member_write", True)
+
+        assert "alerts:write" in member.get_scopes()
+
+        organization.update_option("sentry:alerts_member_write", False)
+
+        assert "alerts:write" not in member.get_scopes()
```

Comment:
Just one extra part to this test: Update the member to be an admin and verify that the setting doesn't break the scope for them.

Reply:
oooh shoot yea good catch